how-we-work/how-to-upgrade-dependencies.md

```diff
@@ -0,0 +1,75 @@
+# How to Upgrade Dependencies
+
+Once a month at the beginning of the month, the oncall engineer should visit the current versions of package dependencies that we have for both the frontend (`npm`) and the backend (`pip`) for `corpora-data-portal` and `cellxgene`. We do this so that we keep up to date with any security vulnerabilities that may exist in older versions and so that we don't end up too out-of-date with improvements that occur in our dependencies causing forward-incompatibility. The sections below describe the policies and instructions for each subset of our product suite.
+
+As a general rule please do not directly import the latest version of the downstream package that contains the vulnerability. For example, if we import package `A` and package `A` depends on package `B` which depends on package `C` which contains a vulnerability, please do not directly import the latest version of package `C` directly into our package. If a newer version of package `A` exists where they have addressed the vulnerability, directly upgrade `A`. Otherwise, wait until there is a new one available.
+
+For an overview of current dependency issues we have in cellxgene Explorer and cellxgene Portal, you can check the two corresponding projects in [Snyk](https://app.snyk.io/org/cellxgene). Note that you should take the findings with a grain of salt as some findings may not have remediation and therefore may continue to exist after everything else has been upgraded. It's a good place to double check your work though.
```

Comment:
you should also look at Dependabot alerts in github.  We should not clear them unless:
* they are fully resolved
* they are not relevant

Reply:
E.g., https://github.com/chanzuckerberg/corpora-data-portal/security/dependabot

Reply:
Done